pkg/strace/internal/binary/binary_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package binary
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+type ts struct {
+	I1 int8
```

Comment:
I find these variable names hard to read ; ) -- but it easily guessable they are just random field names like `foo`, and intention here is to make them short.